Documentation/ApiOverview/SiteHandling/SiteSettings.rst

```diff
@@ -0,0 +1,53 @@
+.. include:: ../../Includes.txt
+
+.. _sitehandling-settings:
+
+=============
+Site Settings
+=============
+
+It is possible to define a `settings` block in a site's :file:`config.yml` which
+can be accessed both in backend and frontend via the site object.
+
+Additionally, these settings are available as "constants" in both TSConfig
+and TypoScript templates making it possible to configure for example site-wide
+storage PIDs which can be used in both frontend and backend.
+
+Adding Site Settings
+====================
+
+Add a `settings` block to the :file:`config.yml`:
+
+.. code-block:: yaml
+
+   settings:
+      categoryPid: 658
+      styles:
+         content:
+            loginform:
+               pid: 23
+
+.. note::
+
+   This example shows how to fill two constants of EXT:felogin via site settings.
```

Comment:
Actually _one_ constant, `categoryPid` is custom after all ...

Reply:
sometimes... I annoy myself - fixed, thanks!